editor/src/components/canvas/canvas-strategies/strategies/convert-to-absolute-and-move-strategy.tsx

```diff
@@ -278,10 +278,10 @@ function convertToAbsoluteAndMoveStrategyFactory(setHuggingParentToFixed: SetHug
             },
           })
 
-          return strategyApplicationResult([
-            ...absoluteMoveApplyResult.commands,
-            strategyIndicatorCommand,
-          ])
+          return strategyApplicationResult(
+            [...absoluteMoveApplyResult.commands, strategyIndicatorCommand],
+            'rerender-all-elements',
```

Comment:
looks like this strategy forgot to set elementsToRerender, we should fix it!

Reply:
Added comment about follow up in 38ec355d5.